src/utils/getCustomAPIWalletAllowed.ts

```diff
@@ -0,0 +1,23 @@
+import axios from "axios";
+import { CustomAPIWalletAllowed } from "../types";
+
+export const getCustomAPIWalletAllowed = async (
+    apiUrl: string,
+    walletAddress: string,
+): Promise<boolean> => {
+  let walletAllowedRes;
+  const finalApiUrl: string = apiUrl.replace("$(wallet)", walletAddress);
+  
+  try {
+    walletAllowedRes = (
+      await axios.get(finalApiUrl)
+    ).data as CustomAPIWalletAllowed;
+
+    return walletAllowedRes.allowed;
+
+  } catch (e) {
+    console.log(e);
```

Comment:
Let's do console.error here:
```suggestion
    console.error(e);
```

I realize I made this mistake for `getRandomEarthTokens.ts` and `getKnowhereTokens.ts` as well